jobs/create-version-branch.js

```diff
@@ -62,12 +62,16 @@ module.exports = async function (
   //
   // See this issue for details: https://github.com/greenkeeperio/greenkeeper/issues/506
 
-  const moduleLockFiles = ['npm-shrinkwrap.json']
-  const projectLockFiles = ['package-lock.json', 'yarn.lock']
-  const hasModuleLockFile = _.some(_.pick(repository.files, moduleLockFiles))
-  const hasProjectLockFile = _.some(_.pick(repository.files, projectLockFiles))
-  const usesGreenkeeperLockfile = _.some(_.pick(repository.packages['package.json'].devDependencies, 'greenkeeper-lockfile'))
+  function isTrue (x) {
```

Comment:
Could you please just add a quick comment why this is necessary, I’m sure we’ll have all forgotten in 6 months :D